### ISSUE TITLE
Annotate after code along.

### ISSUE THREAD

**Issue by gaand** [ga-wdi-boston/js-reference-types#6]
To enhance code along understanding

**Comment by jrhorn424** [comment]
Annotate during code-along?

**Comment by jrhorn424** [comment]
Related to #2?

**Comment by gaand** [comment]
Not while coding, but as part of code along, yes..  Talk about code as it is written and then go back and get the developers to help annotate the code written.  This may relateto catch-ups.

**Comment by jrhorn424** [comment]
I believe I did these exercises as a `catch-up-along` (:smile:) when I lead this training. I can't remember when I did the annotations, though, if at all.

**Comment by jrhorn424** [comment]
FYI @payne-chris-r add a note to README so we can close?

**Comment by RealWeeks** [comment]
@gaand okay to close?

**Comment by gaand** [comment]
No. My suggestion  is not what appears in the read me for this commit.

**Comment by RealWeeks** [comment]
Is this more of a delivery note?

**Comment by gaand** [comment]
Partly? What's in the readme suggests that annotation will happen as the code along proceeds.  I want an explanation as the code is written and then asking the developers to say what the annotations should be.

Demos can be annotate as you go.

**Comment by jrhorn424** [comment]
> Demos can be annotate as you go.

This sounds like a good policy.
